src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,93 @@
+package lotto.domain;
+
+import static lotto.constants.ErrorCode.DUPLICATED_LOTTO_NUMBER;
+import static lotto.constants.ErrorCode.INVALID_LOTTO_SIZE;
+import static lotto.constants.LottoRule.LOTTO_MAX_SIZE;
+import static lotto.constants.Message.SEPARATOR_REGEX;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import lotto.validator.InputValidator;
+
+public class Lotto {
+    private final List<LottoNumber> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validate(numbers);
+        this.numbers = sortByAscent(numbers);
+    }
+
+    // TODO: 하나의 생성자로 처리할 수 있는 방법은 없을까?
+    public Lotto(String number) {
+        List<Integer> numbers = splitToList(number);
+        validate(numbers);
+        this.numbers = sortByAscent(numbers);
+    }
```

Comment:
생성자를 하나만 사용하려고 하시는 이유가 있을까요?
지금처럼 `String`과 `List`를 전달받는 두 가지 방법으로 모두 `Lotto` 객체를 생성할 수 있다면 충분히 괜찮은 코드가 될 것 같다는 것이 저의 생각이에요!
다만, 동일한 로직이 두 가지 생성자에 중복으로 존재한다면 유지보수 측면에서 좋지 않을 것 같으니 정적 팩토리 메서드 패턴으로 여러 가지 값들을 받아와서 동일한 생성자를 호출하도록 할 수 있지 않을까요?
```suggestion
    // TODO: 하나의 생성자로 처리할 수 있는 방법은 없을까?
    public static Lotto from(String number) {
        List<Integer> numbers = splitToList(number);
        return new Lotto(numbers); 
    }
```